From 8c606fb4f6e8bb852d48a3e1228f5ea33bdb308d Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 11 Apr 2017 14:55:39 +0800
Subject: [PATCH] arm: dts: rk3288-evb-rk818: fix tshut prority to HIGH

Depends on hardware design, the PWR_HOLD should be high
for board with rk818 pmic.

Change-Id: Ib18f530d435f08ac98fbf6a9481eb483b7fadece
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-rk818.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-rk818.dts b/arch/arm/boot/dts/rk3288-evb-rk818.dts
index 861ea4658c76..0ac1fe591dec 100644
--- a/arch/arm/boot/dts/rk3288-evb-rk818.dts
+++ b/arch/arm/boot/dts/rk3288-evb-rk818.dts
@@ -327,6 +327,11 @@
 	power-supply = <&vcc_lcd>;
 };
 
+&tsadc {
+	rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
+	rockchip,hw-tshut-polarity = <1>; /* tshut polarity 0:LOW 1:HIGH */
+};
+
 &pinctrl {
 	lcd {
 		lcd_en: lcd-en  {
-- 
2.35.3

